cove_bods/templates/cove_bods/additional_checks_table.html

```diff
@@ -119,10 +119,10 @@
         {% elif additional_check.type == 'person_birth_year_too_late' %}
             <tr>
                 <td>
-                    {% trans 'This Person Statement has a birthday that is to late.' %}
+                    {% blocktrans %}<code>birthDate</code> value is invalid. The year is not far enough in the past. Check that the date is correct and well formatted.{% endblocktrans %}
```

Comment:
I think this check is only looking for dates that are in the future (e.g. a person who is 1 day old would be accepted) in which case 

```suggestion
                    {% blocktrans %}<code>birthDate</code> value is invalid. The date is in the future. Check that the date is correct and well formatted.{% endblocktrans %}
```